### ISSUE TITLE
Scan of first vaccination also display title "Vollständiger Impfschutz"

### ISSUE THREAD

**Issue by Danfro** [arnef/coronaapp#3]
When scanning the [official test qr codes](https://github.com/eu-digital-green-certificates/dgc-testdata/tree/5cec4e540f21e9886a34f5bc9875b95d50542b3e/DE), both qr codes are labelled "Vollständiger Impfschutz".

With only the first vaccination it should better be labelled "Teilweiser Impfschutz" or such.

**Comment by Danfro** [comment]
Hmm. Looks like scanning another code, that is not displayed, only Erika Mustermann. A delete scanned code feature would be great.

But, I know. This is the first release and I am sure you are working on those things already. :angel:

**Comment by arnef** [comment]
> Hmm. Looks like scanning another code, that is not displayed, only Erika Mustermann. 

It's a horizontal scroll view. you should see all scanned certificates by swiping left/right

**Comment by Danfro** [comment]
Yes, I found that out. But there still was only Erika. :-)

Although I actually only managed to scan a code other than Erika once. Seems to need a huge amount of luck. Maybe my printed codes are too small?

**Comment by arnef** [comment]
> Seems to need a huge amount of luck. Maybe my printed codes are too small?

That could be an issue. my printed codes has a size of 4x4cm and they can be scanned without any problem (with bq4.5)

**Comment by Danfro** [comment]
Nope. My code is 5 x 5 cm. For the number of detail I consider that small. :-)

I tried on my E5, which should have a similar camera as the E4.5. Today with daylight I managed to scan the codes, but it was still not easy.

In our German Telegram group other users report too that scanning of codes does not work or only very cumbersome. Scanning the codes with Tagger app works flawless and much faster. So something is different here. But this should be another issue.
